code/game/mecha/working/tree.dm

```diff
@@ -0,0 +1,37 @@
+/obj/mecha/working/tree
+	desc = "This special exosuit is fuelled by living wood. This allows it to be a bit lighter and nimble, but it is weaker than metal."
+	name = "\improper Murmuring Wood"
+	icon_state = "murmuring_wood"
+	silicon_icon_state = "murmuring_open"
+	step_in = 1.5
+	max_integrity = 150
+	lights_power = 0
+	deflect_chance = 0
+	armor = list("melee" = 25, "bullet" = 20, "laser" = 0, "energy" = 0, "bomb" = 40, "bio" = 0, "rad" = 20, "fire" = 100, "acid" = 100)
```

Comment:
Why is a wooden mech fireproof?

Reply:
left over from plastic mech (which probably should not have been fireproof or acidproof either!)